harvard-lts/CURIOSity
# 12-14-2022 Code Changes

**12-14-2022 Code Changes**
* * *


# What does this Pull Request do?
Brings over softserv code to our repo including:
- oai gem version bump
- docker ignore change
- safari arrow fix
- Adding more default fields
- Hiding exhibits from homepage checkbox
- Adding sql client
- Fixing masked role

# How should this be tested?

These have all been test on softserv -- we can deploy this to dev if you'd like @phil-plencner-hl before merging. Also can pull locally and rebuild and make sure things work (arrow does not work for me so I defer to you on that one)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
